packages/extender/src/registry/composeServices.js

```diff
@@ -6,8 +6,8 @@ function composeServices() {
                 let previous = acc[key];
                 if (typeof previous === 'function' && typeof value === 'function') {
                     // If function, override the function but pass the previous one as the last parameter
-                    acc[key] = () => {
-                        value.apply(this, [...arguments, previous]);
+                    acc[key] = function () {
```

Comment:
Why did you transform the lambda into a function?
Can you add a comment in the code to explain why, please?

Reply:
`// Do not convert this function to a lambda otherwise the arguments will not be coming from the right context`